console-ui/src/pages/ConfigurationManagement/ConfigurationManagement/ConfigurationManagement.js

```diff
@@ -1009,16 +1009,28 @@ class ConfigurationManagement extends React.Component {
     }
   }
 
+  goLogin() {
```

Comment:
Why change the goLogin implementation?

Reply:
method goLogin miss references，i just make a copy from other js-file.